flink-connectors/flink-connector-hbase-2.2/src/test/java/org/apache/flink/connector/hbase2/util/HBaseTestBase.java

```diff
@@ -263,11 +262,13 @@ private static Put putRow(
         put.addColumn(
                 Bytes.toBytes(FAMILY4),
                 Bytes.toBytes(F4COL1),
-                Bytes.toBytes(timestampToInternal(f4c1)));
+                Bytes.toBytes(DateTimeUtils.toInternal(f4c1)));
         put.addColumn(
-                Bytes.toBytes(FAMILY4), Bytes.toBytes(F4COL2), Bytes.toBytes(dateToInternal(f4c2)));
+                Bytes.toBytes(FAMILY4), Bytes.toBytes(F4COL2), Bytes.toBytes(toInternal(f4c2)));
```

Comment:
```suggestion
                Bytes.toBytes(FAMILY4), Bytes.toBytes(F4COL2), Bytes.toBytes(DateTimUtils.toInternal(f4c2)));
```
or above use also the static import